recipes/clean-up.rb

```diff
@@ -0,0 +1,7 @@
+
+handler_directory = ::File.join(Chef::Config[:file_cache_path], 'handler')
+
+directory handler_directory do
+  recursive true
+  action :delete
+end
```

Comment:
I see what you're doing now :)  However, I suppose I don't understand what the requirement is to cause us to remove the chef handler ruby library code.
To me it smells fishy because it seems like it's akin to running an app that deletes all traces of itself after execution.  I can think of two potential concerns that arise from that:
- It's not in the spirit of transparency - we're running code on nodes and then deleting that code from the nodes.
- Deleting the handler code from the node could make troubleshooting harder since it requires version correlation against a git repo or chef server to determine what actual code was running when the handler fired.  Having the code on intact on the node gives you a 100% assurance that you know what executed.    

If this moves forward, I recommend adding the standard header to the recipe which gets generated from `chef generate recipe clean-up`  One last thing on this commit, just me perhaps, but using `-` in recipe names is not consistent with the requirement that cookbooks are not allowed to have `-` in the name.

```
#
# Cookbook Name:: audit
# Recipe:: clean-up
#
# Copyright (c) 2016 The Authors, All Rights Reserved.
```